praw/models/reddit/subreddit.py

```diff
@@ -218,7 +218,13 @@ def banned(self):
 
         .. code-block:: python
 
-           reddit.subreddit("SUBREDDIT").banned.add("NAME", ban_reason="...")
+           reddit.subreddit("SUBREDDIT").banned.add("NAME", ban_message="str", ban_reason="str", duration=days, note="str")
```

Comment:
```suggestion
           reddit.subreddit("SUBREDDIT").banned.add("NAME",
               ban_message="You have been banned",
               ban_reason="Bannable action",
               duration=5,
               note="Link to comment: ...")
```
Breaking it up into one argument per line is the way to go. Also, this would normally throw a lint error, but we have long line errors disabled. I think that we should also use actual values instead of the types.